frontend/src/components/media_search.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+    import { routeToPage } from '../utils'
+    import NoResults from '../components/no_results.svelte';
+
+    const SHOWN_PROVIDERS: number = 5;
+    const SHOW_BUTTON_AMOUNT: number = 21;
+    const IMG_URL: string = 'https://image.tmdb.org/t/p/original/';
+	const LOW_RES_IMG_URL: string = 'https://image.tmdb.org/t/p/w500/';
+
+    export let media
+    export let providerAmounts: number[]
+    export let currentCountry: string
+    export let currentProviders
+    export let mediaStartAmount: number
+    export let currentMediaAmount: number
+    export let input: string
+    export let currentGenres: []
+    export let search
+
+    const changeMediaAmount = (buttonElement: string) => {
+        currentMediaAmount = buttonElement === 'loadmore' ? currentMediaAmount + SHOW_BUTTON_AMOUNT : currentMediaAmount - SHOW_BUTTON_AMOUNT;
+        search()
+    }
+
+</script>
+
+{#if media.hits}
+    {#if media.hits.length > 0}
```

Comment:
Redundant?

Reply:
Both checks are needed but made a if that checks both